### DISCUSSION TITLE
dos response time hysteresis on the way down affects fan step down  ?

### DISCUSSION THREAD

**Discussion by DrMental** [Rem0o/FanControl.Releases#1953] [Q&A] [answered]
![image](https://github.com/Rem0o/FanControl.Releases/assets/140925436/4e3d2723-1a25-46c4-ae94-21528951d4be)

![image](https://github.com/Rem0o/FanControl.Releases/assets/140925436/66dcb4a6-966c-442d-9b28-d2fd0b1749bd)

**Comment by Rem0o** [comment, marked as answer]
both are independent settings.
